docker/DOCKERFILE

```diff
@@ -0,0 +1,11 @@
+LABEL maintainer=omelisra123@gmail.com
+#https://hub.docker.com/r/patwoz/expo-cli
+#baseline of this DOCKERFILE come from this repo
+FROM patwoz/expo-cli:3 
```

Comment:
Move this line to the top if not this error is happening
https://stackoverflow.com/questions/50452860/error-response-from-daemon-no-build-stage-in-current-context